sdk/tables/aztables/zc_table_pagers.go

```diff
@@ -29,6 +30,49 @@ type TableEntityQueryResponsePager interface {
 	Err() error
 }
 
+type StructEntityQueryResponsePager interface {
+	NextPage(context.Context) bool
+	PageResponse() StructQueryResponseResponse
+	Err() error
+}
+
+type StructQueryResponseResponse struct {
```

Comment:
Why does this start with `Struct`?

Reply:
Oh this isnt related to the testing framwork right?

Reply:
right - It's some experimentation I was doing with customizing table query responses to serialize to customer provide structs rather than the default `map[string]interface{}`

Reply:
We should probably talk about naming for this later in the public surface area review.

Reply:
Didnt take a hard look at it now.